### DISCUSSION TITLE
能不能介绍一下模型的使用步骤？（Can you introduce the steps for using the model? ）

### DISCUSSION THREAD

**Discussion by old-xno** [linzepore/xeno_rasa#2] [Q&A]
能不能介绍一下模型的使用步骤？（Can you introduce the steps for using the model? ）

**Comment by linzepore** [comment]
完整的步骤如[README.md](https://github.com/linzepore/xeno_rasa#readme)文档所见，简单来说需要以下操作：
1. 安装并启动docker
2. 拉取镜像
3. 下载模型文件
4. 启动容器的同时执行模型启动的命令、挂载模型文件
5. （可选）快捷键 `Ctrl` + `Q` + `P` 让容器后台运行
